geofront/backends/cloud.py

```diff
@@ -83,6 +83,19 @@ class CloudRemoteSet(collections.abc.Mapping):
     :type alias_nameer:
         :class:`~typing.Callable`\ [[:class:`libcloud.compute.base.Node`],
                                     :class:`str`]
+    :param addresser: A function to get the address for the given node.
+                      :attr:`Node.public_ips
+                      <libcloud.compute.base.Node.public_ips>`
+                      is used by default.
+    :type addresser:
+        :class:`~typing.Callable`\ [[:class:`libcloud.compute.base.Node`],
+                                    :class:`str`]
+    :param filter: A function to decide if the given node should be included as
+                   a remote or not.  By default it checks if the node has any
+                   public IP.
+    :type filter:
+        :class:`~typing.Callable`\ [[:class:`libcloud.compute.base.Node`],
+                                    :class:`bool`]
```

Comment:
Added two options should be also logged using [`.. versionchanged:: 0.5.0` directive][1] below.

[1]: http://www.sphinx-doc.org/en/stable/markup/para.html#directive-versionchanged